src/memory/view.rs

```diff
@@ -71,6 +71,17 @@ macro_rules! memory_view {
                         Ok(view[offset + index].get())
                     }
                 }
+
+                pub fn each(&self) {
+                    let view = self.memory.view::<$wasm_type>();
+
+                    let mut offset = self.offset;
+                    while offset < view.len() {
```

Comment:
I think we can write:

```
for nth in self.offset..view.len() {
    let value = view[nth].get() as i64;
    VM::yield_object(Integer::from(value))
}
```

It's more Rust idiomatic, and it avoids misusing the stop condition.

Reply:
I fixed.